packages/panda-san/src/form/input-group.scss

```diff
@@ -0,0 +1,7 @@
+.input-group {
```

Comment:
just curious, is input-group a container for fields/inputs ? If so, was the margin-top meant for spacing between input-groups or inputs?

Reply:
```html
<div className="input-group>
  <label>
  <input className="input">
</div>
```
OR
```html
<label className="input-group">
  <input className="input">
</label>
```

Thiughts?

Reply:
ah, ok ok